kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/launch-dev.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+cd ..
+MVN_ARG_LINE=()
+
+for arg in "$@"
+do
+    case "$arg" in
+        *)
+            MVN_ARG_LINE+=("$arg")
+        ;;
+    esac
+done
+
+startDateTime=`date +%s`
+
+# check that Maven args are non empty
+if [ "$MVN_ARG_LINE" != "" ] ; then
+    mvnBin="mvn"
+    if [ -a $M3_HOME/bin/mvn ] ; then
+       mvnBin="$M3_HOME/bin/mvn"
+    fi
+    echo
+    echo "Running maven build on available projects (using Maven binary '$mvnBin')"
+
+    "$mvnBin" -v
+    echo
+    projects=( "MYSERVICE_NAME_MARKER-model" "MYSERVICE_NAME_MARKER-kjar" "MYSERVICE_NAME_MARKER")
+
+    for suffix in "${projects[@]}"; do
+
+        for repository in $suffix;  do
+        echo
+            if [ -d "$repository" ]; then
+                echo "==============================================================================="
+                echo "$repository"
+                echo "==============================================================================="
+
+                cd $repository
+
+                "$mvnBin" "${MVN_ARG_LINE[@]}"
+                returnCode=$?
+
+                if [ $returnCode != 0 ] ; then
+                    exit $returnCode
+                fi
+
+                cd ..
+                fi
+
+        done;
+    done;
+    endDateTime=`date +%s`
+    spentSeconds=`expr $endDateTime - $startDateTime`
+    echo
+    echo "Total build time: ${spentSeconds}s"
+
+else
+    echo "No Maven arguments skipping maven build"
+fi
+
+echo "Launching the application in development mode - requires connection to controller (workbench)"
+pattern="MYSERVICE_NAME_MARKER"
```

Comment:
should be enough to use

```
pattern="*-service"
```

Reply:
I think for archetype alone this cant be done (assumes users name app *-service"
*-kjar and *-model are ok I think. changed it for that